src/main/java/com/microsoft/azure/functions/worker/handler/WorkerInitRequestHandler.java

```diff
@@ -18,6 +18,12 @@ String execute(WorkerInitRequest request, WorkerInitResponse.Builder response) {
         response.putCapabilities("WorkerStatus", "WorkerStatus");
         response.putCapabilities("RpcHttpBodyOnly", "RpcHttpBodyOnly");
         response.putCapabilities("RpcHttpTriggerMetadataRemoved", "RpcHttpTriggerMetadataRemoved");
+        WorkerMetadata.Builder workerMetadataBuilder = WorkerMetadata.newBuilder();
+        workerMetadataBuilder.setRuntimeName("Java");
```

Comment:
would suggest we use lowercase for consistency with other workers & the worker config
```suggestion
        workerMetadataBuilder.setRuntimeName("java");
```

Reply:
updated.